src/main/webapp/js/query.js

```diff
@@ -233,14 +235,15 @@ $("input, textarea").blur(function() {
 
 // TODO: make this more seamless
 //Loads the results page
-function resultsPage(name, rating, photoUrl) {
-    fetch(`../results.html`)
-        .then((html) => html.text())
+async function resultsPage(name, rating, photoUrl) {
+    return fetch(`../results.html`)
+        .then((response) => response.text())
         .then((html) => {
             document.getElementById("page-container").innerHTML = html;
             document.getElementById("pick").innerText = name;
             document.getElementById("rating").innerText = rating;
             loadImage(photoUrl);
+            return "";
```

Comment:
will returning "" cause issues? Just want to make sure...

Reply:
removed